test/lightning/credentials_test.exs

```diff
@@ -355,6 +256,245 @@ defmodule Lightning.CredentialsTest do
     end
   end
 
+  describe "create_credential/1" do
+    test "suceeds with raw schema" do
+      valid_attrs = %{
+        body: %{"username" => "user", "password" => "pass"},
```

Comment:
please add an `integer` and a `boolean` to this example raw credential and assert that they are not dumped as strings